server/routes/index.js

```diff
@@ -0,0 +1,9 @@
+const
+    routes          = require('express').Router(),
+    userController  = require('../controllers/user.controller')
+
+routes.route('/api/user')
```

Comment:
I suggest to create a file for each type of route (user.routes.js, product.routes.js, order.routes.js)